src/SonarScanner.MSBuild.PostProcessor/PostProcessor.cs

```diff
@@ -39,7 +39,6 @@ public class PostProcessor : IPostProcessor
     private readonly ITargetsUninstaller targetUninstaller;
```

Comment:
`System.IO;` does not seem to be used anymore.

Reply:
It's used within the preprocessor directive `#if NETFRAMEWORK` for `Path.Combine`